### DISCUSSION TITLE
Error when installing addon

### DISCUSSION THREAD

**Discussion by maxwell-01** [tsightler/ring-mqtt#566] [Support Q&A] [answered]
Hello,

I have HAOS installed and have followed the instructions to install the addon. I have 2 issues:

- The addon didn't initially show up after adding the repo, I had to find a direct link on reddit. After opening the direct link it started showing up. The link used: `http://homeassistant.local:8123/hassio/addon/03cabcc9_ring_mqtt/info`
- I cannot install the addon, please see error below:

`Can't install tsightler/ring-mqtt:5.1.3: 500 Server Error for http+docker://localhost/v1.41/images/tsightler/ring-mqtt:5.1.3/json: Internal Server Error ("readlink /mnt/data/docker/overlay2/l: invalid argument")`

I'm wondering if this is not the correct version of the addon, perhaps as a result from step 1.

Any help greatly appreciated.

**Comment by maxwell-01** [comment, marked as answer]
Turns out this was an issue with Home Assistant and not with this addon at all, running the command `ha su repair` solved the problem.

**Reply by tsightler** [reply]
Thanks for posting the resolution.  In the end this addon is already installed by ~5000 users of HAOS, so it makes sense that any issue is unlikely to be with the addon or repo itself.
